gnat2goto/COMPILING.md

```diff
@@ -68,6 +68,29 @@ through to final analysis by CBMC.
 
 Instructions for running the regression test suite are described in [testsuite/gnat2goto/README.md](testsuite/gnat2goto/README.md)
 
+# Running and updating missing features tests
+
+Currently gnat2goto is lacking support for a number of Ada language features.
+The CI system runs gnat2goto on a number of open source Ada projects to track
+changes in feature coverage. These tests can be run locally by performing the
+following commands:
+
+- `$ export PATH=/opt/gnat/bin:/path/to/gnat2goto/bin:${PATH}`
+- `$ export GPR_PROJECT_PATH=/opt/gnat/lib/gnat`
+- `$ cd gnat2goto/experiments`
+- `$ ./CI-list_unsupported.sh`
+
+This script will checkout a number of Ada projects, then for each project it will
+run gnat2goto and summarize the unsupported features used by the project. The
+summary is then compared against a 'golden' reference summary.
```

Comment:
Might make the 'golden' here a link to an explanation, like [this](https://ro-che.info/articles/2017-12-04-golden-tests)?

Reply:
I thought this was a relatively well known term (it is, afterall, the same type of testing we use for things like CBMC regression tests, gnat2goto regression tests, etc), but I can dig out some sort of reference if you want for a followup PR.

Reply:
To be honest, I didn't know it was an established term. I mean, I could more or less imagine what it meant, but if it wasn't for Hannes' link I wouldn't know.

Reply:
OK, I'll add a reference then :-)